drivers/shared/executor/executor.go

```diff
@@ -469,6 +469,14 @@ func (e *UniversalExecutor) Shutdown(signal string, grace time.Duration) error {
 		proc.Kill()
 	}
 
+	// Wait for process to exit
+	select {
+	case <-e.processExited:
+	case <-time.After(time.Second * 15):
```

Comment:
nitpick: I find it easier to read when the scaler comes first, e.g. `15 * time.Second`, than `time.Second * 15`